pkg/bosh/converter/docker_image.go

```diff
@@ -3,28 +3,31 @@ package converter
 import (
 	corev1 "k8s.io/api/core/v1"
 
-	"code.cloudfoundry.org/quarks-job/pkg/kube/controllers/extendedjob"
+	"code.cloudfoundry.org/quarks-utils/pkg/config"
 	"code.cloudfoundry.org/quarks-utils/pkg/names"
 )
 
 // operatorDockerImage is the location of the operators own docker image
-var operatorDockerImage = ""
+var operatorDockerImage string
+var operatorImagePullPolicy corev1.PullPolicy
```

Comment:
I guess we will set invalid value in container.ImagePullPolicy over GetOperatorImagePullPolicy() if `operatorImagePullPolicy`  == "". How about set default value?

Reply:
Done in https://github.com/cloudfoundry-incubator/cf-operator/pull/656/commits/624e0cf08357e77bd37ffde20b0b21fc4c8d5794.